app/views/volunteer_ops/index.html.erb

```diff
@@ -1,3 +1,14 @@
+<%= form_tag(search_volunteer_ops_path, :id => "organisation_search", :method => "get") do %>
+    <div class="flex-row">
+      <div class="left-side">
+        <%= label_tag(:q, "Search Text", class: 'field-label') %>
+      </div>
+      <div class="right-side">
+        <%= text_field_tag(:q, @query) %>
```

Comment:
note that we have some untested functionality here, i.e. that the query string is visible after the each search